modules/cnf-topology-aware-lifecycle-manager-precache-user-spec-images.adoc

```diff
@@ -0,0 +1,259 @@
+    - aws
+    - vsphere
+  additionalImages: <4>
+    - quay.io/exampleconfig/application1@sha256:3d5800990dee7cd4727d3fe238a97e2d2976d3808fc925ada29c559a47e2e1ef
+    - quay.io/exampleconfig/application2@sha256:3d5800123dee7cd4727d3fe238a97e2d2976d3808fc925ada29c559a47adfaef
+    - quay.io/exampleconfig/applicationN@sha256:4fe1334adfafadsf987123adfffdaf1243340adfafdedga0991234afdadfsa09
+----
+<1> The `platformImage`, `operatorsIndexes`, and the `operatorsPackagesAndChannels` fields can be configured to override the default {cgu-operator}-derived values. These fields are automatically populated from the policies of the managed clusters if they are not specified.
+<2> Specifies the minimum required disk space on the cluster in Gigabytes. If unspecified, {cgu-operator} defines a default value for {product-title} images. The disk space field must include an integer value and the storage unit. For example: `40 GiB`, `200 MB`, `1 TiB`.
+<3> Specifies the list of images that can be excluded from the cluster version update.
+<4> Specifies the list of additional images to pre-cache.
+
+[id="talm-prechache-user-specified-images-preparing-crs_{context}"]
+== Creating the custom resources for pre-caching
+
+You must create the `PreCachingConfig` CR before or concurrently with the `ClusterGroupUpgrade` CR.
+
+. Create the `PreCachingConfig` CR with the list of additional images you want to pre-cache.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: PreCachingConfig
+metadata:
+  name: exampleconfig
+  namespace: default <1>
+spec:
+[...]
+  spaceRequired: 30Gi <2>
+  additionalImages:
+    - quay.io/exampleconfig/application1@sha256:3d5800990dee7cd4727d3fe238a97e2d2976d3808fc925ada29c559a47e2e1ef
+    - quay.io/exampleconfig/application2@sha256:3d5800123dee7cd4727d3fe238a97e2d2976d3808fc925ada29c559a47adfaef
+    - quay.io/exampleconfig/applicationN@sha256:4fe1334adfafadsf987123adfffdaf1243340adfafdedga0991234afdadfsa09
+----
+<1> The `namespace` must be accessible to the hub cluster.
+<2>  It is recommended to set the minimum disk space required field to ensure that there is sufficient storage space for the pre-cached images.
+
+. Create a `ClusterGroupUpgrade` CR with the `preCaching` field set to `true` and specify the `PreCachingConfig` CR created in the previous step:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu
+  namespace: default
+spec:
+  clusters:
+  - sno1
+  - sno2
+  preCaching: true
+  preCachingConfigRef:
+  - name: exampleconfig
+    namespace: default
+  managedPolicies:
+    - du-upgrade-platform-upgrade
+    - du-upgrade-operator-catsrc-policy
+    - common-subscriptions-policy
+  remediationStrategy:
+    timeout: 240
+----
+
+[WARNING]
+====
+Once you install the images on the cluster, you cannot change or delete them.
+====
+
+. When you want to start pre-caching the images, apply the `ClusterGroupUpgrade` CR by running the following command:
```

Comment:
The step numbering here seems to indicate a new procedure. Or else it should be step 3  in the ongoing procedure.
If it's a new procedure it needs to be introduced in some way. Otherwise the mark-up needs to be corrected so that this is labelled step #3